src/nvim/misc1.c

```diff
@@ -1617,30 +1606,23 @@ int del_chars(long count, int fixpos)
   return del_bytes(bytes, fixpos, TRUE);
 }
 
-/*
- * Delete "count" bytes under the cursor.
- * If "fixpos" is TRUE, don't leave the cursor on the NUL after the line.
- * Caller must have prepared for undo.
- *
- * return FAIL for failure, OK otherwise
- */
-int 
-del_bytes (
-    long count,
-    int fixpos_arg,
-    int use_delcombine                  /* 'delcombine' option applies */
-)
+/// Delete "count" bytes under the cursor.
+/// If "fixpos" is true, don't leave the cursor on the NUL after the line.
+/// Caller must have prepared for undo.
+///
+/// @param  count           number of bytes to be deleted
+/// @param  fixpos_arg      leave the cursor on the NUL after the line
+/// @param  use_delcombine  'delcombine' option applies
+///
+/// @return FAIL for failure, OK otherwise
+int del_bytes(int count, bool fixpos_arg, int use_delcombine)
```

Comment:
I wonder if `int` is the correct type here. The count here is not a vim count but a count of column bytes, so maybe `colnr_T` or perhaps `size_t` as it is a byte count. Regardless, there are a few more call sites to update (no warnings as `ops.c` are not -Wconversion yet)